packages/terra-toolbar/src/Toolbar.jsx

```diff
@@ -0,0 +1,40 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import classNames from 'classnames/bind';
+import styles from './Toolbar.module.scss';
+
+const cx = classNames.bind(styles);
+
+const propTypes = {
+  /**
+   * Content alignment. Align to start, end, or center.
+   */
+  align: PropTypes.oneOf(['start', 'end', 'center']),
+  /**
+   * Items to be displayed in toolbar.
```

Comment:
Can we add collapsible menus in here? Dropdown buttons? etc?

Reply:
Technically yes. Though in the future I believe there will be a toolbar designed specifically for used with the collapsible menu. I can add examples of items to be used in the description.